src/native/minipal/com/minipal_com.h

```diff
@@ -0,0 +1,138 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifndef MINIPAL_COM_H
+#define MINIPAL_COM_H
+
+#include "comtypes.h"
+
+#ifdef __cplusplus
+    extern "C"
+    {
+#endif // __cplusplus
+
+//
+// Memory allocators
+//
+LPVOID PAL_CoTaskMemAlloc(SIZE_T);
+void PAL_CoTaskMemFree(LPVOID);
+
+//
+// Strings
+//
+
+// This macro is used to standardize wide character string literals used in .NET.
+#ifdef DNCP_WINDOWS
+    #define W(str)  L ## str
+#else
+    #define W(str)  u ## str
+#endif
+
+size_t PAL_wcslen(WCHAR const*);
+int PAL_wcscmp(WCHAR const*, WCHAR const*);
+WCHAR* PAL_wcsstr(WCHAR const*, WCHAR const*);
```

Comment:
Can we maintain the `minipal_` prefix convention? This API duplicates functionality already present in `src/coreclr/minipal`. It might be more appropriate to place it in `src/native/minipal/strings.h` and dedup.

Reply:
Resolved and extracted to https://github.com/dotnet/runtime/pull/108999